### ISSUE TITLE
If attorney using UpToCode, generate Word files, not just PDF

### ISSUE THREAD

**Issue by nonprofittechy** [nonprofittechy/docassemble-HousingCodeChecklist#143]

**Comment by nonprofittechy** [comment]
Depends on #104

**Comment by nonprofittechy** [comment]
Actually this is already done--the DOCX is in the "download" link while "view" is the PDF.
